README.md

```diff
@@ -170,3 +126,82 @@ Now, for any new html page, we need to do the following:
 {% endblock %}
 
 ```
+
+# Deployment to Fly.io
+
+## Install FlyCtl
+
+To work with the Fly platform, you first need to install Flyctl, a command line interface that allows you to do everything from creating an account to deploy to Fly.
+  - Linux:
+    ```  
+    curl  -L https://fly.io/install.sh | sh
+    ```
+  - OSX:
+    ```
+    brew install flyctl
+    ```
+  - Windows:
+    ```
+    iwr https://fly.io/install.ps1 -useb
+    ```  
+    Here is the [Installation Guilde](https://fly.io/docs/hands-on/install-flyctl/)
+
+
+Next authenticate with your fly.io account:
+  ```
+  fly auth login
+  ```
+
+- To make sure everything is working well:    
+  ```
+  fly apps list
+  ```
+The output of the above command will be empty table since you have no apps launched yet
+
+
+## Configure the Project
+
+### Environment Variables
+
+We shouldn't store secrets in source code, so utilizing environmental variables is needed.
+- Run the following from the root of your project
+  ```
+  python3 set_env.py
+  ```
+
+### Deploy App
+
+In this step the app is going to be launched to fly.io.
+- Create and configure the app  
+  ```
```

Comment:
Again. I thought I asked that there is a step earlier in the config where you give your username, like rohel or turian or whatever. It validates it's only numbers, letters, and dashes. And then it automates choosing the app name:

audio-discrimination-framework-USERNAME-prod

and later


audio-discrimination-framework-USERNAME-staging